internal/federationdomain/resolvedprovider/resolvedgithub/resolved_github_provider_test.go

```diff
@@ -50,4 +62,17 @@ func TestFederationDomainResolvedGitHubIdentityProvider(t *testing.T) {
 		UpstreamUsername: "fake-upstream-username",
 		GitHub:           &psession.GitHubSessionData{},
 	}))
+	redirectURL, err := subject.UpstreamAuthorizeRedirectURL(
+		&resolvedprovider.UpstreamAuthorizeRequestState{
+			EncodedStateParam: "encodedStateParam12345",
+			PKCE:              "pkce6789",
+			Nonce:             "nonce1289",
+		},
+		"https://localhost/fake/path",
+	)
+	require.NoError(t, err)
+	require.Equal(t,
+		"?client_id=clientID12345&redirect_uri=https%3A%2F%2Flocalhost%2Ffake%2Fpath%2Fcallback&response_type=code&state=encodedStateParam12345",
```

Comment:
GitHub does not document the `response_type=code` param for their authorize endpoints. Did you test if they will ignore it? Or can you cause it to be excluded from our URL?

Reply:
I confirmed via manual testing that GitHub will ignore this param.